shard/src/ext/settings_cog.py

```diff
@@ -306,16 +343,21 @@ async def roleids(self, ctx):
         await ctx.channel.send(embed=lem.get_embed())
 
     @commands.command()
-    async def settings(self, ctx):
+    async def settings(self, ctx, tag="general"):
         '''Open an interactive settings dialog'''
         settings = self.bot.settings[ctx.guild]
         if ctx.author.id not in settings.admins_ids:
             await ctx.channel.send('nope, sorry')
             return
 
-        msg = await ctx.channel.send(embed=await self.get_embed(ctx, settings))
+        settings_with_tag = filter(lambda s: tag in s.tags, self.settings_elements)
+        if settings_with_tag == 0:
```

Comment:
`len`

Reply:
hmm, I guess this works because an empty array is falsy LOL.

Reply:
Will add len though